src/screens/GovernanceCall/GovernanceCallBanner.js

```diff
@@ -0,0 +1,191 @@
+/* eslint-disable i18next/no-literal-string */
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2021 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import * as React from 'react';
+import { Linking } from 'react-native';
+import styled from 'styled-components/native';
+import { useTranslationWithPrefix } from 'translations/translate';
+import {
+  addHours,
+  differenceInMinutes,
+  differenceInHours,
+  differenceInDays,
+  isAfter,
+  isBefore,
+} from 'date-fns';
+
+// Components
+import Image from 'components/Image';
+import Text from 'components/core/Text';
+
+// Utils
+import { fontStyles, spacing } from 'utils/variables';
+import { reportErrorLog } from 'utils/common';
+import { mapFromDocumentDataToString } from 'utils/prismic';
+import { formatDate } from 'utils/date';
+
+// Services
+import * as Prismic from 'services/prismic';
+
+// assets
+const smartWalletIcon = require('assets/icons/smart-wallet-migrate.png');
+
+const TYPE_GOVERNANCE_CALL = 'governance-calls';
+const DATE_FORMAT = 'dd.MM';
+const TIME_WITH_TIMEZONE_FORMAT = 'HH:mm OOO';
+
+const GovernanceCallBanner = () => {
+  const { t } = useTranslationWithPrefix('governanceCall');
+  const [title, setTitle] = React.useState(t('banner.title'));
+  const [description, setDescription] = React.useState('');
+  const [scheduledCallTime, setScheduledCallTime] = React.useState(0);
+  const [governanceCallLink, setGovernanceCallLink] = React.useState('');
+  const [showPermanently, setShowPermanently] = React.useState(false);
+
+  React.useEffect(() => {
+    async function fetchGovernanceCallData() {
```

Comment:
Can you move this out as `useCallback` and only leave the actual call on `useEffect`?